napari/utils/_tests/test_theme.py

```diff
@@ -98,6 +101,11 @@ def test_rebuild_theme_settings():
     settings.appearance.theme = "another-theme"
 
 
+@pytest.mark.skip(
+    os.getenv('CI') and sys.version_info < (3, 9),
```

Comment:
Ok, what do I have wrong here ? 

On ubuntu 3.9 I get:
```
napari/utils/_tests/test_theme.py .....ssssss.  
```

Which is obviously not waht I want.

Reply:
I think you want 

https://docs.pytest.org/en/6.2.x/reference.html#pytest-mark-skipif
`@pytest.mark.skipif(`